index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+    <script src="./script.js" defer></script>
+    <title>Zamalek official site</title>
+  </head>
+  <body class="fonts">
+    <header>
+      <nav class="topnav">
+        <div class="burger">
+          <a id="close-menu">&times;</a>
+          <ul class="burger-menu">
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#">Videos</a></li>
+            <li><a href="#">Matches</a></li>
+            <li><a href="#">Teams</a></li>
+            <li><a href="#">Shop</a></li>
+          </ul>
+        </div>
+        <div class="navbars">
+          <ul class="nav1">
+            <li><a class="white" href="#"><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+            <li><a class="white" href="#"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
+            <li><a class="white" href="#">English</a></li>
+            <li><a class="white" href="#">My Page</a></li>
+            <li><a class="white move" href="#">Logout</a></li>
+          </ul>
+          <div class="nav2">
```

Comment:
- [ Optional ] You should add more padding on the left and right side of the navbar as in the design.
- [Optional] The gap between the logo and the nav menu items seems wider in your implementation as compared to the design.
#### Design
![image](https://user-images.githubusercontent.com/44057863/169667383-d7976491-a6d6-4205-bca0-5223bb464c98.png)

#### Impl
![image](https://user-images.githubusercontent.com/44057863/169667367-a7ffd983-d3e0-4fc4-9aac-67906b2a1ec6.png)